md/views.py

```diff
@@ -31,6 +32,37 @@ def search(request):
 class AgencyList(ListView):
```

Comment:
MD AgencyList and NC AgencyList are almost identical, right?  It "should be easy" to share almost all of the new implementation with an intermediate class.  That could have been done before, but now there is a lot more code.  The hardest part is deciding where to put it.  `traffic_stops/base_views.py` perhaps?